analyses/compare-gistic/02-GISTIC-tidy-data-prep.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "GISTIC tidy data preparation"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: Chante Bethell for ALSF CCDL
+date: 2020
+---
+
+This notebook will tidy the GISTIC data files required to compare the GISTIC's gene level calls with our gene level calls in `analyses/focal-cn-file-prepartion/results`. The required GISTIC files include `all_lesions.conf_90.txt`, `amp_genes.conf_90.txt`, and `del_genes.conf_90.txt` (these file are stored in `analyses/run-gistic/results`). 
+
+The purpose of this notebook is to prepare GISTIC data in a format that can be compared to our focal CN files (both on a gene level and cytoband level). 
+
+## Usage
+
+This notebook is intended to be run via the command line from the top directory
+of the repository as follows:
+
+```
+Rscript -e "rmarkdown::render('analyses/compare-gistic/02-GISTIC-tidy-data-prep.Rmd', clean = TRUE)"
+```
+
+## Set up
+
+```{r}
+library(tidyverse)
+```
+
+### Files and Directories
+
+```{r}
+# Path to input directory
+gistic_data_dir <- file.path("..", "run-gistic", "results")
+
+# Path to output directory
+results_dir <- "results"
+
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+```
+
+### Unzip GISTIC results
+
+```{r}
+# Unzip and set up GISTIC results folders and files.
+cohort_gistic_dir <-
+  file.path(gistic_data_dir, "pbta-cnv-consensus-gistic")
+cohort_gistic_zip <-
+  file.path(gistic_data_dir, "pbta-cnv-consensus-gistic.zip")
+
+if (!dir.exists(cohort_gistic_dir)) {
+  unzip(cohort_gistic_zip,
+    exdir = data_dir
+  )
+}
+
+lgat_gistic_dir <-
```

Comment:
I put this step in there because the 02 notebook is not dependent on the 01 notebook, so if someone wanted to run the 02 notebook independently they can. However, I am not sure how important having this option is here (especially considering that there is an upcoming notebook that _will_ be dependent on the 02 notebook).